components/clm/src/biogeophys/SoilHydrologyMod.F90

```diff
@@ -1513,7 +1562,8 @@ subroutine Drainage(bounds, num_hydrologyc, filter_hydrologyc, num_urbanc, filte
                 end if
              endif
 
-#if (defined HUM_HOL)
+! bsulman: Does MARSH need this deep_seep stuff?
```

Comment:
I'm not sure that it needs a deep seep since we want to the water to flow out through column 2 anyway instead of being through the bottom of the column